apps/storybook/src/TransferList.stories.tsx

```diff
@@ -0,0 +1,242 @@
+/* eslint-disable react/jsx-key */
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  TransferList,
+  TransferListProps,
+  IconButton,
+} from '@itwin/itwinui-react';
+
+import {
+  SvgChevronLeft,
+  SvgChevronRight,
+  SvgChevronLeftDouble,
+  SvgChevronRightDouble,
+} from '@itwin/itwinui-icons-react';
+
+export default {
+  component: TransferList,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+  },
+  title: 'Core/TransferList',
+} as Meta<TransferListProps>;
+
+export const Basic: Story<TransferListProps> = (args) => {
```

Comment:
Does it make sense to show examples of transferlist with only one listbox? I'd imagine this component is pretty much always used with two listboxes.